.github/stale.yml

```diff
@@ -0,0 +1,31 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
+
+# Number of days of inactivity before a stale Issue or Pull Request is closed
+daysUntilClose: 14
```

Comment:
On the other hand, if we introduce this bot now, we might want to give more time for users to respond to issues. Thus, making me change the default of 7 days to 14 for days until close. I think we may adjust the values as we go, but at the beginning it might be better to start off high.